### PR TITLE
doc: contributing guide including build instructions

### LINKED ISSUE #22: [FEAT] Add local build instructions
This is very interesting, i would like to contribute, but i am unsure on how to build the patched chromium locally for testing. Can you add a simple description?

### PR DESCRIPTION
**Pull request by 34N0**
fixes #22

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+`git clone --depth 1 --recurse-submodules https://github.com/secureblue/hardened-chromium.git && cd hardened-chromium`
+
+Download chromium-%{version}-clean.tar.xz from the Fedora's server:
+
+`rpkg --path ./chromium sources`
+
+#### Harden
+Copy the patches inside the source folder:
+
+`cp vanadium_patches/* ./chromium && cp patches/* ./chromium`
```
Comment (qoijjj) [inline CONTRIBUTING.md:62]: This looks great, thanks! 

The only nit I have is that the steps up to this point can be accomplished by running copr_script.sh I think?

Comment (34N0) [reply CONTRIBUTING.md:62]: yes but copr_script copies all the files to the root of the project for copr to build. this is makes it unusable for local development. The steps are the same but everything happens inside the chromium folder.

Comment (qoijjj) [reply CONTRIBUTING.md:62]: makes sense, thanks

File: patches/set-default-secure-dns-mode-automatic.patch [hunk 2]
```diff
@@ -7,7 +7,7 @@ index 27526ac0c574b..0680d2912d381 100644
                       SecureDnsConfig::ParseMode(mode_str)
                           ? std::string(mode_str)
 -                         : SecureDnsConfig::kModeOff);
-+                         : SecureDnsConfig::kAutomatic);
```
Comment (qoijjj) [inline patches/set-default-secure-dns-mode-automatic.patch:10]: looks like you need a rebase :)

Comment (qoijjj) [reply patches/set-default-secure-dns-mode-automatic.patch:10]: let's see what happens if I squash

**Review by qoijjj — APPROVED** [review]

**Pull request merged**